airflow/opt/providers/etna/etna/tests/cassettes/test_metis_files_etl/test_metis_files_etl_e2e.yaml

```diff
@@ -15504,24 +15504,24 @@ interactions:
       string: '{"type":"file","id":58306,"parent_id":51335,"node_name":"MVIR1-HS15-D0PBMC1-RSQ1_S3_L001_R1_001.fastq.gz","updated_at":"2020-09-10
         15:04:46 +0000","created_at":"2020-09-10 15:04:46 +0000","file_hash":"5af58f96c94f25141a2dd2529453665b","archive_id":"Ik4bYgcSJEDcr6cvkn_dd1xwb0iG3QzCSLNC_lSC9lCRAYLWSm9KwC3zgOmvZazOynkruPzL4TaJC2ZwP2rQv4dJoAJP4homDqxGd_pZVqrcsqMip3LeDv0PJq9AyYeS0WSw6bQL-A"}
 
-        {"type":"file","id":58307,"parent_id":51335,"node_name":"MVIR1-HS15-D0PBMC1-RSQ1_S3_L001_R2_001.fastq.gz","updated_at":"2020-09-10
-        15:09:50 +0000","created_at":"2020-09-10 15:09:50 +0000","file_hash":"a98fa07eb6246b1f4c31579a7d32bf5c","archive_id":"im_uCsoNSNplfWadC-Ulyka_OGlGIH451ZJM32dqqC1b_44h3VLHYVScz1_nGkqhuT3m9-cYiqnv16RON0Y1Nevr0C8zPh2MXeSf80KH7Tgoqi2vWjKECr54_224EuNwYwxjthRPsA"}
-
         {"type":"file","id":58308,"parent_id":51336,"node_name":"MVIR1-HS15-D0PBMC1-RSQ3_S37_L002_R1_001.fastq.gz","updated_at":"2020-09-10
         15:18:15 +0000","created_at":"2020-09-10 15:18:15 +0000","file_hash":"51fe26a68df4047bea5b8c01e7f319f1","archive_id":"2IRLXTFo727MpqYAEK8SVVYREWs7oo-ok66cTG0VIEy5KbgsIbqkkVfgws-BOk9Qj5qULB9k6KPvXTJXCFxO44bCbBsa0zOcTuKzNDCohhjW6gPB6PlW_iw-rzjSlXgoNWP9Td3MCw"}
 
+        {"type":"file","id":58307,"parent_id":51335,"node_name":"MVIR1-HS15-D0PBMC1-RSQ1_S3_L001_R2_001.fastq.gz","updated_at":"2020-09-10
```

Comment:
Sorry, I got lazy and didn't want to re-record the entire cassette....so just manually edited a bit to simulate the test condition we see that causes the issue...

Reply:
That's fair.